contrib/scrooge/src/python/pants/contrib/scrooge/tasks/scrooge_gen.py

```diff
@@ -78,6 +78,13 @@ def register_options(cls, register):
             type=dict,
             help="Registered target types.",
         )
+        register(
+            "--allowlist-compiler-args",
```

Comment:
Like the comment below, could we make this name a bit more descriptive? Specifically, `allowlist` doesn't tell me _what_ we allow them to do. Something like:
```suggestion
            "--unchecked-compiler-args",
```